docs/advanced-features/workflows.md

```diff
@@ -461,243 +725,22 @@ such as `object`, `array`, and compound `enum` types.
 This allows for flexible and extensible schema definition, and hence
 validation, of complex metadata schemas to any depth.
 
-> Quilt currently uses the Draft 4 Json Schema where tuples are
```

Comment:
This block is not outdated. `prefixItems` keyword still is not supported. Draft 7 was released in 2018

I think, there was a typo: "Draft 4" → "Draft 7"

The reason, we mention what we don't support, is because some customer used `prefixItems`

Reply:
Oh, I found it